dapps/marketplace/src/hoc/withGrowthCampaign.js

```diff
@@ -44,6 +45,12 @@ function withGrowthCampaign(
                     {...props}
                     growthEnrollmentStatus={enrollmentStatus}
                     growthCampaigns={get(data, 'campaigns.nodes') || []}
+                    growthCampaignsLoading={
```

Comment:
This should be just `walletLoading` since `walletLoading = !networkStatus || loading || networkStatus === 1`

Reply:
Once the wallet loads, the growth campaigns are requested and `networkStatus ` and `loading` variables are related to campaigns query and not wallet query.

Reply:
Sorry I missed that it was different query :/

Perhaps I should get some sleep.

Reply:
hehe no worries, it means that you are giving it a thorough code review. I really appreciate that thanks!